app/ai/voice/agents/breeze_buddy/managers/recording.py

```
@@ -0,0 +1,103 @@
+                "original_url": provider_recording_url,
+            },
+        )
+
+        if gcs_url:
+            logger.info(f"Successfully uploaded recording to GCS: {gcs_url}")
+            await update_lead_call_recording_url(call_id, gcs_url)
+        else:
+            logger.error(f"Failed to upload recording to GCS for call_id: {call_id}")
+            await update_lead_call_recording_url(call_id, provider_recording_url)
+
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)